README.md

```diff
@@ -46,48 +18,3 @@ Example- for our directory structure it is:
 ```sh
  python3 main.py /video 
 ```
-* To compile the C++ version of the code with openCV the command is:
```

Comment:
why this whole paragraph is stripped? @YacobBY Can you please explain? the README looks way too much small with little to no information about the project.

Reply:
That's a mistake should be put back, is for my personal version.

Reply:
Didn't realize it would commit changes I made after submitting the version to you

Reply:
please resolve this issue if its a mistake